src/internet_identity/src/assets.rs

```diff
@@ -132,5 +133,11 @@ fn get_assets() -> [(&'static str, &'static [u8], ContentEncoding, ContentType);
             ContentEncoding::Identity,
             ContentType::ICO,
         ),
+        (
+            "/.well-known/ic-domains",
+            include_bytes!("../../../dist/ic-domains"),
```

Comment:
how about inlining the content here? I think that might make things easier

Reply:
Ah, of course. I didn't even think of that. 👍